### ISSUE TITLE
Is this project active?

### ISSUE THREAD

**Issue by GrbavaCigla** [DeflatedPickle/NeoTextureEdit#1]

**Comment by DeflatedPickle** [comment]
@GrbavaCigla, it is! But I've been a bit discoraged with what I was working on (`obj-models`) as I ran into a roadblock I couldn't figure out.

**Comment by GrbavaCigla** [comment]
How to build this, I am on archlinux?

**Comment by DeflatedPickle** [comment]
@GrbavaCigla, since I ported it to use Gradle for building, you can use that. I'll add some building instructions when I come back to the project.

**Comment by GrbavaCigla** [comment]
Any updates? Also i dont know java, any instructions how to build it?

**Comment by DeflatedPickle** [comment]
Nope, I think I have given up. There are better alternatives elsewhere, even web-based ones.

But if you really want to build it, I haven't added tasks for it. But it's Gradle, so all you have to do is invoke `run`.
